### PR TITLE
Metrics: Adds setting for turning off total stats metrics

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
Update of total stats (stat_totals_*) metrics shouldn't be made when metrics endpoint is disabled.
Generation of total stats metrics is made each minute and can add quite high load on a database so for those that still want metrics endpoint enabled, but without total stats metrics, we introduce a new setting `disable_total_stats` to configure that. 

**Which issue(s) this PR fixes**:
Ref #19137

**Special notes for your reviewer**:

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -593,8 +593,10 @@ enabled = true
 #################################### Internal Grafana Metrics ############
 # Metrics available at HTTP API Url /metrics
 [metrics]
-enabled           = true
-interval_seconds  = 10
+enabled              = true
+interval_seconds     = 10
+# Disable total stats (stat_totals_*) metrics to be generated
+disable_total_stats = false
```
Comment (kylebrandt) [inline conf/defaults.ini]: I think having enabled and disable directives in the same block might be confusing. Maybe `enable_total_stats` with a default of `true`?

**Review by kylebrandt — COMMENTED** [review]
https://github.com/grafana/grafana/blob/master/docs/sources/installation/configuration.md#metrics needs updating.

File: conf/defaults.ini (hunk 1 above)
Comment (marefr) [reply conf/defaults.ini]: We want boolean settings to be default false. But can be discussed in this case.

Comment (kylebrandt) [reply conf/defaults.ini]: Ah that makes sense then.

**Review by kylebrandt — APPROVED** [review]
LGTM once .md doc is updated

**Review by torkelo — APPROVED** [review]

**Pull request merged**